docker/hoodie/hadoop/rahil.sh

```diff
@@ -0,0 +1,19 @@
+docker build base -t yihua/hudi-hadoop_3.1.0-base
```

Comment:
rename file name to something else, and replace the alias `yihua`. Instead take first arg from script

Reply:
@yihua Instead im thinking that the alias should just be `apachehudi` similar to what it has been.

Reply:
Yes, it is a temporary work-around to use my alias for fast iteration.  I've uploaded the latest docker images based on this PR to `apachehudi` docker hub account.